test/dummy/db/migrate/20120508165529_create_tables.rb

```diff
@@ -6,7 +6,7 @@ def self.up
 
       ## Database authenticatable
       t.string :email,              null: false, default: ''
-      t.string :encrypted_password, null: false, default: ''
+      t.string :encrypted_password, null: true, default: ''
```

Comment:
Not sure if you need this change. I can back it out if not.

Reply:
Let's stick with the default (`null: false`) for the migration. Thanks!

Reply:
Done :)